python/ray/rllib/train.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import ray
+import ray.rllib.policy_gradient as pg
+import ray.rllib.evolution_strategies as es
+import ray.rllib.dqn as dqn
+import ray.rllib.a3c as a3c
+import simplejson
```

Comment:
I'm not totally sure what the right convention is here, but the ray imports should probably be separate and placed below